Hazel/src/Hazel/ImGui/ImGuiLayer.cpp

```diff
@@ -48,7 +48,10 @@ namespace Hazel {
 
 		// Setup Platform/Renderer bindings
 		ImGui_ImplGlfw_InitForOpenGL(window, true);
-		ImGui_ImplOpenGL3_Init("#version 410");
+		ImGui_ImplOpenGL3_Init("#version 450");
+//		ImGui_ImplOpenGL3_Init("#version 430 core");
```

Comment:
What's the point of leaving old code commented. Hazel will only support OpenGL 4.5

Reply:
Removed in https://github.com/TheCherno/Hazel/pull/82/commits/833380b615d9c1c2c3d1122b7fda3207a1aaa678.